source/includes/_configuration-options.md

```diff
@@ -32,11 +32,29 @@ const configurationOptions = {
     hideSimilarProducts: false,
     useCustomCss: true
   },
-  useDeepLinking: false
+  useDeepLinking: false,
+  customLabels: {
+    buy: 'custom button label'
+  }
 }
 ```
 They will be described in the following sections.
 
+***
+
+## Primary Color
+
+If you'd like to unify our design with your own color-theme, you can use our primary color configuration option to change the color of `buy` buttons in Product and indicator of items number in Cart. You only need to make sure to use the right hex color code.
+
+```javascript
+const tipserSdk = TipserSDK("59e86b79b8f3f60a94ecd26a", {primaryColor: "#5F9F9F"});
+```
+
+[![](primary-color.png)](/images/primary-color.png)
+
+* If your primary color is bright, you may consider changing also the text color for elements like buttons via CSS overrides of element's specific class [(examples)](#customizing-tipser-elements-styles). 
```

Comment:
@mateuszsikora This what we could get rid of, while doing this auto discord of good contrast color for text.

Reply:
Yeah, secondary color auto-discovery sounds like a great idea!

Reply:
Secondary color auto-discovery sounds like a nice idea at first sight. But on the other hand, imo there is a high risk to discover a color that is completely not compatible with POS color theme, so they will need to change it manually anyway. In that case, is it worth the effort?

Reply:
I would guess yes! I think it's low hanging fruit, welcome one, that will help us avoid making css changes by POS.